components/forms/Label/Label.tsx

```diff
@@ -30,13 +30,7 @@ export const Label = (props: LabelProps): React.ReactElement => {
 
   const childrenElements = (
     <>
-      {children}{" "}
-      {required ? (
-        <span data-testid="asterisk" aria-hidden>
-          *
-        </span>
-      ) : null}
-      {required ? <i className="visually-hidden">{t("required-field")}</i> : null}
+      {children} {required && <span data-testid="required">({t("required-field")})</span>}
```

Comment:
So it might be better to keep the `i` element as in voice over I now hear required instead of "Required Field" as was before. It's better if its required field as its much clearer to the  user

Reply:
Fair enough.

I looked into just adding a `label` to the `<span>`, but some screen readers may ignore the labels on non-semantic HTML elements, of which `<span>` is one. That could be why it was set up the way it was originally anyway.